src/ExperimentTable.js

```diff
@@ -95,6 +95,13 @@ class ExperimentTable extends React.Component {
     })
   }
 
+  componentDidMount(){
```

Comment:
You don’t need to add `componentDidMount`. Just initialise the state fields `searchQuery` and `searchedColumnIndex` and the props will naturally flow down to `TableContent`.

Reply:
`columnNumber` is not a stateful variable. We need to specify which column the `species` value is searching.

Reply:
But `searchedColumnIndex` is.